packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -272,7 +273,9 @@ const MultiSelect = React.forwardRef(function MultiSelect(
             />
           )}
           <span id={fieldLabelId} className={`${prefix}--list-box__label`}>
-            {label}
+            {itemToString && enableSelectionLabel && selectedItems.length > 0
```

Comment:
Would you mind pulling this out into a helper function? We try to not inline conditionals like when they are checking for multiple conditions, thank you!

Reply:
Done